test/cqlpy/test_describe.py

```diff
@@ -1488,19 +1487,28 @@ def __exit__(self, exc_type, exc_val, exc_tb) -> None:
                 self.cql.execute(f"DROP SERVICE LEVEL {make_identifier(sl, quotation_mark='"')}")
 
 class ServiceLevel:
-    def __init__(self, name: str, timeout: int|None = None, wl_type: str|None = None):
+    default_shares_value = 1000
```

Comment:
I'm pretty sure this one needs to be 1000, please see the `get_create_stmt` method and see the comment about the `replace_default_shares` parameter.

We use this method in DESCRIBE tests: when describing a service level, Scylla will include the `WITH shares = ...` part unconditionally, even if you didn't provide it when creating the service level, and in that case the number of shares will be equal to 1000. Our service levels tests check this property.

Reply:
But this is exactly my point - if your test always sets the value and then checks the value - why does it need to be 1000? It can be set to 987 and then reading it will retrieve 987, not 1000. The only place you should read 1000 is when you didn't override the default.

Reply:
It's not that the test always sets the shares, it's Scylla who writes the default shares into its internal tables. For example, if the client sends:

```
CREATE SERVICE LEVEL x;
```

...the DESCRIBE will return this statement:

```
CREATE SERVICE LEVEL x WITH shares = 1000;
```

Hence the `replace_default_shares` parameter - with `False` it generates the first variant, with `True` - the latter.